sympy/integrals/tests/test_intpoly.py

```diff
@@ -179,6 +179,187 @@ def test_polytope_integrate():
     assert polytope_integrate(cube4, x ** 2 + y ** 2 + x * y + z ** 2) ==\
            S(37) / 960
 
+    #  Test cases from Mathematica's PolyhedronData library
+    octahedron = [[(S(-1) / sqrt(2), 0, 0), (0, S(1) / sqrt(2), 0),
+                   (0, 0, S(-1) / sqrt(2)), (0, 0, S(1) / sqrt(2)),
+                   (0, S(-1) / sqrt(2), 0), (S(1) / sqrt(2), 0, 0)],
+                  [3, 4, 5], [3, 5, 1], [3, 1, 0], [3, 0, 4], [4, 0, 2],
+                  [4, 2, 5], [2, 0, 1], [5, 2, 1]]
+
+    assert polytope_integrate(octahedron, 1) == sqrt(2) / 3
+
+    great_stellated_dodecahedron =\
+        [[(-0.32491969623290634095, 0, 0.42532540417601993887),
+          (0.32491969623290634095, 0, -0.42532540417601993887),
+          (-0.52573111211913359231, 0, 0.10040570794311363956),
+          (0.52573111211913359231, 0, -0.10040570794311363956),
+          (-0.10040570794311363956, -0.3090169943749474241, 0.42532540417601993887),
+          (-0.10040570794311363956, 0.30901699437494742410, 0.42532540417601993887),
+          (0.10040570794311363956, -0.3090169943749474241, -0.42532540417601993887),
+          (0.10040570794311363956, 0.30901699437494742410, -0.42532540417601993887),
+          (-0.16245984811645317047, -0.5, 0.10040570794311363956),
+          (-0.16245984811645317047,  0.5, 0.10040570794311363956),
+          (0.16245984811645317047,  -0.5, -0.10040570794311363956),
+          (0.16245984811645317047,   0.5, -0.10040570794311363956),
+          (-0.42532540417601993887, -0.3090169943749474241, -0.10040570794311363956),
+          (-0.42532540417601993887, 0.30901699437494742410, -0.10040570794311363956),
+          (-0.26286555605956679615, 0.1909830056250525759, -0.42532540417601993887),
+          (-0.26286555605956679615, -0.1909830056250525759, -0.42532540417601993887),
+          (0.26286555605956679615, 0.1909830056250525759, 0.42532540417601993887),
+          (0.26286555605956679615, -0.1909830056250525759, 0.42532540417601993887),
+          (0.42532540417601993887, -0.3090169943749474241, 0.10040570794311363956),
+          (0.42532540417601993887, 0.30901699437494742410, 0.10040570794311363956)],
+         [12, 3, 0, 6, 16], [17, 7, 0, 3, 13],
+         [9, 6, 0, 7, 8], [18, 2, 1, 4, 14],
+         [15, 5, 1, 2, 19], [11, 4, 1, 5, 10],
+         [8, 19, 2, 18, 9], [10, 13, 3, 12, 11],
+         [16, 14, 4, 11, 12], [13, 10, 5, 15, 17],
+         [14, 16, 6, 9, 18], [19, 8, 7, 17, 15]]
+    #  Actual volume is : 0.163118960624632
+    assert polytope_integrate(great_stellated_dodecahedron, 1) -\
+        0.163118960624632 < 1e-15
```

Comment:
You need to use the `abs(x-y) < eps` idiom, not `x-y < eps`. The latter would be satisfied, e.g., for x=1, y=5.

Reply:
15 digits are printed, but I would still lower `eps` to something like 1e-12 or so, just in case.